llama_parse/base.py

```diff
@@ -34,159 +34,136 @@
 _DEFAULT_SEPARATOR = "\n---\n"
 
 
-class LlamaParse(BasePydanticReader):
-    """A smart-parser for files."""
-
-    api_key: str = Field(
-        default="",
-        description="The API key for the LlamaParse API.",
-        validate_default=True,
-    )
-    base_url: str = Field(
-        default=DEFAULT_BASE_URL,
-        description="The base URL of the Llama Parsing API.",
-    )
-    result_type: ResultType = Field(
-        default=ResultType.TXT, description="The result type for the parser."
-    )
-    num_workers: int = Field(
-        default=4,
-        gt=0,
-        lt=10,
-        description="The number of workers to use sending API requests for parsing.",
-    )
-    check_interval: int = Field(
-        default=1,
-        description="The interval in seconds to check if the parsing is done.",
-    )
-    max_timeout: int = Field(
-        default=2000,
-        description="The maximum timeout in seconds to wait for the parsing to finish.",
-    )
-    verbose: bool = Field(
-        default=True, description="Whether to print the progress of the parsing."
-    )
-    show_progress: bool = Field(
-        default=True, description="Show progress when parsing multiple files."
-    )
-    language: Language = Field(
-        default=Language.ENGLISH, description="The language of the text to parse."
-    )
-    parsing_instruction: Optional[str] = Field(
-        default="", description="The parsing instruction for the parser."
+class LlamaParseParams(BasePydanticReader):
```

Comment:
Lets add a comment to make it clear this class is ONLY for params that are required in the `data` field of requests to parsing